common/scala/src/main/scala/whisk/core/entity/WhiskAction.scala

```diff
@@ -109,6 +109,14 @@ case class WhiskAction(
     require(exec != null, "exec undefined")
     require(limits != null, "limits undefined")
 
+    /** @return true iff action has appropriate annotation. */
+    def hasFinalParamsAnnotation = {
```

Comment:
Want to fold `asBool` https://github.com/openwhisk/openwhisk/pull/1715/files#diff-85adc8c36c3cfcd17be782427443d12eR87 into this pull-request?